addons/survey/models/survey_user.py

```diff
@@ -68,13 +71,25 @@ def _compute_scoring_success(self):
         for user_input in self:
             user_input.scoring_success = user_input.scoring_percentage >= user_input.survey_id.scoring_success_min
 
-    @api.depends('start_datetime', 'survey_id.is_time_limited', 'survey_id.time_limit')
+    @api.depends(
```

Comment:
Not sure, but isn't it easier to have survey_time_limit_reached and session_time_limit_reached ? That way we separate a bit things.

Reply:
I gave it a try, we'll see how it goes.